src/valves.py

```diff
@@ -0,0 +1,66 @@
+#import BoardInterface
+
+from enum import Enum
+
+# Valve: class representing an individual valve on the test stand. Needs access
+# to the BoardInterface code to implement the open() and close() methods.
+
+class Valve:
+
+    def __init__(self, valveID, boardID, closedAngle, openAngle, name):
+        self._valveID = valveID
+        self.boardID = boardID
+        self._closeAngle = closeAngle
+        self._openAngle = openAngle
+        self._lastValue = 0.0
+        self.name = name
+
+    def getValue(self):
```

Comment:
A little thing that would be nice to have that I just thought of:
Since we can't actually read the value of the valve, we should record what the last value we set it to was. Thus, in open/close, we can set `_lastValue` to be the `_openAngle` and `_closedAngle`. We should also check to see if someone is doing something stupid, like opening a valve twice before closing it and log a warning (just use `print()` for now).

Then, change `getValue` to a function called `is_open()`, that returns a boolean saying if the last set value was open or closed. Would you mind updating the LucidChart diagram to reflect this change as well?

For now, assume that valves default to closed. We can change that later after getting feedback from Jack/Lewis.